x-pack/test_serverless/api_integration/test_suites/common/platform_security/api_keys.ts

```diff
@@ -163,6 +163,31 @@ export default function ({ getService }: FtrProviderContext) {
           expect(status).toBe(200);
         });
 
+        it('has_api_keys', async () => {
```

Comment:
__note:__ Thanks for the API integration test! Can we move it to `x-pack/test/security_api_integration/tests/api_keys/has....test` since it's not serverless-specific (in a dedicated file we can check if we return `false` before user has any keys, then create a key, and check again)?

Optionally, we can also test how this API behaves with invalidated and expired keys (e.g. we can create a key with very short expiration date in the test).

Reply:
Sure - I've moved this: 474ed251bca